packages/terra-badge/src/Badge.jsx

```diff
@@ -46,20 +46,16 @@ const Badge = ({
   isReversed,
   text,
   icon,
-  ...additionalProps
+  ...customProps
 }) => {
-  const classes = classNames([
-    'terra-Badge',
-    { [`terra-Badge--${size}`]: size },
-    { [`terra-Badge--${intent}`]: intent },
-    additionalProps.className,
-  ]);
+  const badgeClassNames = classNames(styles.badge,
```

Comment:
Is there a new linting rule that changed this? I kinda like how each property was aligned and on it's own separate line.